src/scripts/browser.js

```diff
@@ -99,6 +99,19 @@ function iOSversion() {
     return [];
 }
 
+function osxVersion() {
+    const matches = /Mac OS X (\d+)[._](\d+)[._]?(\d+)?/i.test(navigator.userAgent);
```

Comment:
Please don't. You should not try to get macOS version from user agent because you cannot. All browsers only reports 10.15 on macOS now because a lot of websites have stupid assumptions that major version of macOS cannot be larger than 10.